go_lib/cloud-data/reconciler.go

```diff
@@ -262,7 +262,7 @@ func (c *Reconciler) instanceTypesReconcile(ctx context.Context) {
 
 	if err != nil {
 		c.updateResourceErrorMetric.WithLabelValues().Set(1.0)
-		c.logger.Errorln("Cannot update cloud data resource. Timed out. See error messages below.")
+		c.logger.Fatalln("Cannot update cloud data resource. Timed out. See error messages below.")
```

Comment:
On the one hand, the decision to crash is a good one, a kube one. But it was done on purpose. Some providers, especially with openstack, like to block the API at a certain threshold of incorrect requests, so we always do not try to discover data from the cloud, do fire the alert and leave the decision to restart to the engineer.

In fact, a good solution would be to crash on requests to the kube-api, and fire the alert on requests to the cloud